internal/quickstart/choose_sdk.go

```diff
@@ -61,6 +64,13 @@ func NewChooseSDKModel(selectedIndex int) tea.Model {
 	}
 }
 
+// initSDKs sets the index of each SDK based on place in list.
+func initSDKs() {
```

Comment:
Do this programmatically instead of explicitly setting an index on each SDK. We use the index to keep track of which SDK was selected so it is highlighted when the user goes back to the list.